src/openms/source/SIMULATION/MSSim.cpp

```diff
@@ -299,6 +299,7 @@ namespace OpenMS
       SignedSize scan_index = distance<SimTypes::MSSimExperiment::ConstIterator>(experiment_.begin(), it_rt);
       pi.setMetaValue("RT_index", scan_index);
       pi.setRT(f.getRT());
+      pi.setMZ(f.getMZ());
```

Comment:
Could someone who knows this code better than me (@timosachsenberg @cbielow @aiche) please confirm that this is always safe? Could there be a case where this PeptideIdentification already has the precursor m/z value set at this point, and this value is different from the feature m/z, so the `pi.setMZ(f.getMZ());` could cause trouble here? Seems safe to me, but I admit I haven't exhaustively tested all possible combinations of MSSimulator's parameters yet ;)